packages/styles/combobox.css

```diff
@@ -111,8 +111,24 @@
   position: relative;
 }
 
+.Combobox .TextFieldWrapper {
+  min-height: 32px;
+  flex-wrap: wrap;
+  gap: var(--space-half);
+  padding-inline: var(--space-smallest);
+  padding-block: var(--space-half);
+}
+
 .Combobox .TextFieldWrapper input[type='text'] {
   margin-right: 1.25rem;
+  flex-grow: 1;
+  width: unset;
+  padding: unset;
+}
+
+.Combobox__selected-tag {
+  margin-left: 0 !important;
```

Comment:
Why `!important`? This should be limited to extremely specific use cases (such as utility classes) where the desired behavior is to take on that style without respect of specificity. There are likely better alternatives that we need to consider before using `!important`.